va/va.go

```diff
@@ -665,18 +665,13 @@ func tlsDial(ctx context.Context, hostPort string, config *tls.Config) (*tls.Con
 	if err != nil {
 		return nil, err
 	}
+	if deadline, ok := ctx.Deadline(); ok {
```

Comment:
Is there ever a normal case in which `!ok`? It seems like we should probably have an audit err log line for the case where we get this far without a `Deadline` on the context to catch future bugs.